.taskcluster.yml

```diff
@@ -6,301 +6,287 @@ policy:
-                              env:
-                                  # run-task uses these to check out the source; the inputs
-                                  # to `mach taskgraph decision` are all on the command line.
-                                  $merge:
-                                      - MOBILE_BASE_REPOSITORY: '${baseRepoUrl}'
-                                        MOBILE_HEAD_REPOSITORY: '${repoUrl}'
-                                        MOBILE_HEAD_REF: '${head_branch}'
-                                        MOBILE_HEAD_REV: '${head_sha}'
-                                        MOBILE_REPOSITORY_TYPE: git
-                                        TASKGRAPH_BASE_REPOSITORY: https://hg.mozilla.org/ci/taskgraph
-                                        TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
-                                        TASKGRAPH_HEAD_REV: ${taskgraph.revision}
-                                        TASKGRAPH_REPOSITORY_TYPE: hg
-                                        REPOSITORIES: {$json: {mobile: "Fenix", taskgraph: "Taskgraph"}}
-                                        HG_STORE_PATH: /builds/worker/checkouts/hg-store
-                                        ANDROID_SDK_ROOT: /builds/worker/android-sdk
-                                      - $if: 'tasks_for in ["github-pull-request"]'
-                                        then:
-                                            MOBILE_PULL_REQUEST_NUMBER: '${event.pull_request.number}'
-                                      - $if: 'tasks_for == "action"'
-                                        then:
-                                            ACTION_TASK_GROUP_ID: '${action.taskGroupId}'  # taskGroupId of the target task
-                                            ACTION_TASK_ID: {$json: {$eval: 'taskId'}}  # taskId of the target task (JSON-encoded)
-                                            ACTION_INPUT: {$json: {$eval: 'input'}}
-                                            ACTION_CALLBACK: '${action.cb_name}'
-                              features:
-                                  taskclusterProxy: true
-                                  chainOfTrust: true
-                              # Note: This task is built server side without the context or tooling that
-                              # exist in tree so we must hard code the hash
-                              image:
-                                  mozillareleases/taskgraph:decision-mobile-44b6b7b4c370220eff56efa8b508aa5157ef9c6e74847c7ecc19d640946ba49e@sha256:4107cbc5e154502529e4d38efa4dc89c05ee54e2cbc6e2e66023e68407502894
+                    maxRunTime: 1800
 
-                              maxRunTime: 1800
+                    command:
+                        - /usr/local/bin/run-task
+                        - '--mobile-checkout=/builds/worker/checkouts/src'
+                        - '--task-cwd=/builds/worker/checkouts/src'
```

Comment:
I think this will still work, but shouldn't these be `vcs` instead of `src`?

Reply:
If you do change them back, don't forget to update the `docker-images` reference later on as well.